mycroft/util/string_utils.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Common string utilities used by various parts of core."""
+
+import re
+
+
+def get_http(uri):
```

Comment:
Are these really heavily used utilities? They seem like unnecessary bloat.

Reply:
I'll mark the http and remove trailing slash functions for removal in the next major release